src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java

```diff
@@ -455,12 +459,35 @@ public boolean parseTownyAdminCommand(String[] split) throws TownyException {
 				giveBonus(StringMgmt.remFirstArg(split));
 
 			} else if (split[0].equalsIgnoreCase("reload")) {
-
-				reloadTowny(false);
-
+				if (split.length == 2) {
+					switch (split[1]) {
+						case "db":
+						case "database":
+							reloadDatabase(false);
+							break;
+						case "config":
+							reloadConfig();
+							break;
+						case "perms":
+						case "permissions":
+							reloadPerms();
+							break;
+						case "language":
+						case "lang":
+							reloadLangs();
+							break;
+						default:
+							player.sendMessage(ChatTools.formatTitle("/ta reload"));
+							player.sendMessage(ChatTools.formatCommand("", "/ta reload", "<database|config|perms|lang>", ""));
+					}
+				} else {
+					player.sendMessage(ChatTools.formatTitle("/ta reload"));
+					player.sendMessage(ChatTools.formatCommand("", "/ta reload", "<database|config|perms|lang>", ""));

```

Comment:
The subcommands should be broken down into 5 options, one subcommand per line the fourth "" is actually meant for translatable help descriptions explaining what each does. Add on an "all" option.